### ISSUE TITLE
Exclude a pattern

### ISSUE THREAD

**Issue by SimonHurst** [glv2/bruteforce-wallet#17]
Hi wondered it's possible to exclude a pattern such as pppp, 1111 or 123 for those of us that don't know much else other than that there are no sequence of repetition all while passing in specific character range.

be great if I've missed something or of this is easier than I think it would be ;)

**Comment by glv2** [comment]
Currently pattern exclusion is not implemented in the program.

**Comment by SimonHurst** [comment]
Yeah I thought that, would be a really great improvement. If I get anywhere on my own I’ll make a PR suspect it may be a little too complex.

**Comment by Lionsara** [comment]
hello, i want to run this on kali linux and dont work, can someone help me?
